src/message_templates/slack_failure.json

```diff
@@ -0,0 +1,98 @@
+{
+	"text": "CircleCI job failed.",
+	"blocks": [
+		{
+			"type": "header",
+			"text": {
+				"type": "plain_text",
+				"text": "Job Failed. :red_circle:",
+				"emoji": true
+			}
+		},
+		{
+			"type": "section",
+			"fields": [
+				{
+					"type": "mrkdwn",
+					"text": "*Job*: ${CIRCLE_JOB}"
+				}
+			]
+		},
+		{
+			"type": "section",
+			"fields": [
+				{
+					"type": "mrkdwn",
+					"text": "*Project*: $CIRCLE_PROJECT_REPONAME"
+				},
+				{
+					"type": "mrkdwn",
+					"text": "*Branch*: $CIRCLE_BRANCH"
+				},
+				{
+					"type": "mrkdwn",
+					"text": "*Workflow*: $CIRCLE_STAGE"
+				},
+				{
+					"type": "mrkdwn",
+					"text": "*Author*: $CIRCLE_USERNAME"
+				},
+				{
+					"type": "mrkdwn",
+					"text": "*Diff Url*: $DIFF_URL"
+				}				
+			],
+			"accessory": {
+				"type": "image",
+				"image_url": "https://assets.brandfolder.com/otz5mn-bw4j2w-6jzqo8/original/circle-logo-badge-black.png",
```

Comment:
Can you download the official assets from https://circleci.com/presskit.zip and just include them in the repo here?

Reply:
I uploaded this to our CDN -- use the link https://cdn.coda.io/external/dev/circle-logo-badge-black.png